index.html

```diff
@@ -3,9 +3,84 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Josh Glantz Codes</title>
+    <link rel="stylesheet" href="style/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;700&display=swap" rel="stylesheet">
```

Comment:
As we noted, you can use `@import` in CSS, but I think this is still the recommended way. That changes when you have a full build system that's compiling and minifying your css/js, but that's a ways out still I think.